Makefile

```diff
@@ -1,17 +1,16 @@
 # The following parameters are required:
-# - GITHUB_ORG
-# - GITHUB_REPO
-# - CIRCLECI_TOKEN
+# - REPOSITORY
+# - CIRCLECI_API_KEY
 #
 
-AWS_ACCESS_KEY_ID := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys | jq -r '.SecretString'| jq -r .access_key_id)
-AWS_SECRET_ACCESS_KEY := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys | jq -r '.SecretString'| jq -r .secret_key)
+GITHUB_ORG := $(shell echo ${REPOSITORY} | cut -d "/" -f 2)
+GITHUB_REPO := $(shell echo ${REPOSITORY} | cut -d "/" -f 3)
 
 run:
 	@echo "Set CIRCLECI environment variables\n"
-	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_ACCESS_KEY_ID", "value":"${AWS_ACCESS_KEY_ID}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
-	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_SECRET_ACCESS_KEY", "value":"${AWS_SECRET_ACCESS_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_ACCESS_KEY_ID", "value":"${AWS_ACCESS_KEY_ID}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_SECRET_ACCESS_KEY", "value":"${AWS_SECRET_ACCESS_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}
```

Comment:
Are AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY passed in as args now?

Reply:
Oops, no. Just missed that they were referenced in there. Added them back in.